.changes/next-release/enhancement-Eventstream-60668.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Eventstream",
+  "description": "The limit for event streams maximum payload size was raised from 16Mb to 24Mb."
```

This is inaccurate since the 16Mb validation doesn't exist currently. I suggested new wording below:
```suggestion
  "description": "Reintroduce a client-side maximum payload size limit, now set to 24Mb."
```

---

Wording was addressed in b1960151a2335f5f87a6a9d44ecb2b7f65c66768